src/spider/storage/mysql/MySqlStorage.cpp

```diff
@@ -1285,6 +1299,10 @@ auto MySqlMetadataStorage::get_ready_tasks(
                         )
                 };
             }
+            // Job id might not be in job_id_to_task_ids if the job's tasks are leased
```

Comment:
Is it possible for job_id to be in job_id_to_task_ids if the job's tasks are leased?

```suggestion
            // Job id will not be in job_id_to_task_ids if the job's tasks are leased
```